makefile

```diff
@@ -173,6 +176,7 @@ distrib: $(TARGET)
 	cp $(MINGW_PATH)/bin/libstdc++-6.dll $(ARCHIVE)/
 	cp $(MINGW_PATH)/bin/libwinpthread-1.dll $(ARCHIVE)/
 	cp $(MINGW_PATH)/bin/zlib1.dll $(ARCHIVE)/
+	cp $(MINGW_PATH)/bin/$(CAPSIPFDLL) $(ARCHIVE)/CAPSImg.dll
```

Comment:
It's actually caught by our new windows continuous build :-) :
https://ci.appveyor.com/project/ColinPitrat/caprice32/build/37/job/sjhyvyp26r9iaera